vck/src/commonMain/kotlin/at/asitplus/wallet/lib/iso/DeviceRequest.kt

```diff
@@ -217,7 +202,7 @@ data class DeviceResponse(
 @Serializable
 data class Document(
     @SerialName("docType")
-    val docType: String,
+    val docType: String, // TODO this is relevant for deserializing the elementValue of IssuerSignedItem
```

Comment:
Remove this comment

Reply:
removed the TODO, but kept the comment

Reply:
But that's wrong, since we're using the namespace now

Reply:
still relevant, isn't it? aren't we using both?

Reply:
I'm pretty sure we use the namespace only, but let's keep it at it ...